packages/cli/test/integration/commands/reset.cmd.test.ts

```diff
@@ -26,7 +26,7 @@ afterAll(async () => {
 	await testDb.terminate();
 });
 
-test('user-management:reset should reset DB to default user state', async () => {
```

Comment:
This test times out a lot more as the test suite grows. disabling it for now. Will re-enable it in https://github.com/n8n-io/n8n/pull/5464

Reply:
I re-enabled it: https://github.com/n8n-io/n8n/pull/9205

It wasn't timing out for me. Let me know if I should not re-enable it.